specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2023-07-01/managedHsm.json

```diff
@@ -1697,6 +1703,45 @@
         }
       },
       "description": "The security domain properties of the managed hsm."
+    },
+    "Identity": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The identity type. Only UserAssigned is currently supported."
+        },
+        "userAssignedIdentities": {
+          "type": "object",
+          "description": "Object containing details of the user assigned identities associated with the resource. The key is the ARM resource identifier of the identity.",
+          "additionalProperties": {
+            "$ref": "#/definitions/UserAssignedIdentity"
+          }
+        }
+      }
+    },
+    "UserAssignedIdentity": {
+      "type": "object",
+      "readOnly": true,
+      "description": "Detail information about the user assigned identity",
+      "properties": {
+        "principalId": {
+          "type": "string"
+        },
+        "clientId": {
+          "type": "string"
+        }
+      }
+    },
```

Comment:
Please check if this type is defined in common types file already. Since this is used across RPs, it should be already defined as a common types.

Reply:
Using common type